src/main/java/com/google/sps/datastore/GetDatastore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class GetDatastore {
+
+    public ArrayList<PCAPdata> dataTable = new ArrayList<PCAPdata>();
```

Comment:
you dont need this variable. you should not maintain this state unless you are using it to cache and avoid looking up the datastore every time.